CHANGELOG.md

```diff
@@ -1,23 +1,64 @@
-x.y.z Release notes (yyyy-MM-dd)
+10.0.0-beta.3 Release notes (yyyy-MM-dd)
```

Comment:
The version is set as part of the release process, not in random PRs adding things to the changelog.